File: primeNumber.py
def is_prime(n):
    if n <= 1:
        return False
    for i in range(2, n//2):
        if n % i == 0:
            return False
    return True

number = 29
if is_prime(number):
    print(f"{number} is prime.")
else:
    print(f"{number} is not prime.")

File: primeNumber.c
#include <stdio.h>

int isPrime(int n){
    if (n <= 1) return 0;
    for (int i = 2; i < n/2; i++){
        if (n%i == 0){
            return 0;
        }
    }
    return 1;
}


int main(){
    int number = 29;
    int check = isPrime(number);
    if (check == 1) printf("%d is a prime number",number);
    else printf("%d is not a prime number",number);
    
    return 0;
}
